### PR TITLE
Enhancing UI Consistency and Accessibility in Button Components with Dark Mode Integration

### PR DESCRIPTION
**Pull request by gkatrakazas**
This Pull Request aims to enhance the user interface consistency and accessibility across our application's button components. By redefining and unifying the design elements of our buttons, we strive to provide a seamless and intuitive user experience. Here's what we've updated:

- **Standardization of Button Types**: We've defined a core set of button types, such as 'Primary', 'Secondary', and 'Cancel', ensuring a consistent approach in their usage.
- **Uniform Styling**: All buttons now share common styling attributes, including color schemes and hover effects, promoting a uniform appearance across the application.
- **Accessibility Improvements**: Enhanced keyboard navigation is now more intuitive with consistent focus styles. We've ensured that all buttons are reachable using the Tab key and maintain a logical focus sequence.
- **Dark Mode Support**: We've extended our design improvements to include a dark mode setting, ensuring our button inputs and general layout adapt elegantly to both light and dark themes.

These changes not only improve the visual consistency of our buttons but also advance the accessibility of our application, making it more inclusive for all users.

### CODE REVIEW

File: src/components/Sidebar.js [hunk 1]
```diff
@@ -141,6 +145,7 @@ const Sidebar = ({ isOpen, toggle }) => {
 
 						<li
 							onClick={handleLogout}
+							tabIndex={0}
 							className={`cursor-pointer flex items-center space-x-2 mb-4 p-2 rounded-r-xl nav-item-animate-hover`}
 						>
```
[the remark this answers is not in the document]
Comment (emlun) [reply src/components/Sidebar.js:150]: Similarly for the `<ul><li>`s in the two diffs below, too.

**Comment by gkatrakazas** [comment]
You're right, and I agree with your points. Please let me know if you notice anything else that might need attention or improvement.

File: src/pages/AddCredentials/AddCredentials.js [hunk 2]
```diff
@@ -150,14 +150,14 @@ const Issuers = () => {
 						style={{ maxHeight: '80vh' }}
 					>
 						{filteredIssuers.map((issuer) => (
-							<li
+							<button
```
Comment (emlun) [inline src/pages/AddCredentials/AddCredentials.js:153]: These `<button>`s are now children of a `<ul>`, perhaps that should be changed to `<div>` or something like that instead?

File: src/pages/SendCredentials/SendCredentials.js [hunk 3]
```diff
@@ -136,14 +136,14 @@ const Verifiers = () => {
 						style={{ maxHeight: '80vh' }}
 					>
 						{filteredVerifiers.map((verifier) => (
-							<li
+							<button
```
Comment (emlun) [inline src/pages/SendCredentials/SendCredentials.js:139]: These `<button>`s are now children of a `<ul>`, perhaps that should be changed to `<div>` or something like that instead?

**Review by emlun — APPROVED** [review]
Looks good! :slightly_smiling_face:

**Review by emlun — APPROVED** [review]

**Review by kkmanos — APPROVED** [review]

**Review by pstamatop — APPROVED** [review]

**Pull request merged**